### DISCUSSION TITLE
How to center all elements in canvas

### DISCUSSION THREAD

**Discussion by brunoocrv** [GrapesJS/grapesjs#5640] [General]
Hello guys, I need to center every blocks added in canvas, but my current tries doesn´t work.

I tried to add the code below to blocks definition:

`
			selectedComponent.addStyle({
				display: 'flex',
				'align-items': 'center',
				'justify-content': 'center'
			})
`

But when I drag and drop the block the component to the canvas, this rules doens´t work

**Comment by artf** [comment]
For such a case providing a reproducible demo might be helpful, there are too many edge cases for "centering elements" in web
